.github/workflows/go.yml

```diff
@@ -34,4 +34,4 @@ jobs:
       run: go build -v .
 
     - name: Test
-      run: go test github.com/googleinterns/ocsp-response-linter/test -v .
+      run: go test ./... -v .
```

Comment:
Just confirming that this is supposed to be relative path vs using the url like before.

Reply:
Yeah since now I have tests in separate directories, I think this is the only way I can get all tests to run.